src/test/java/com/microsoft/sqlserver/jdbc/TimeoutTest.java

```diff
@@ -129,6 +129,17 @@ public void testNestedTimeoutShouldKeepTimerThreadRunning() throws SQLException
         }
     }
 
+    @Test
+    public void testGetClosedTimer() throws SQLServerException, SQLException {
+        try (SQLServerConnection conn = getConnection()) {
+            try {
+                SharedTimer timer = conn.getSharedTimer();
```

Comment:
Fail with Exception not thrown error after this statement if no exception is thrown.